### ISSUE TITLE
Check failed: mdb_status == 0 (2 vs. 0) No such file or directory

### ISSUE THREAD

**Issue by wendaoyuchen** [weiliu89/caffe#378]
I download the models_VGGNet_VOC0712_SSD_300x300.tar.gz and unzip it  into the caffe-ssd/models/,  then I iuput "python home/usrname/caffe-ssd/examples/ssd/score_ssd_pascal.py", the error comes out, "Check failed: mdb_status == 0 (2 vs. 0) No such file or directory"

![default](https://cloud.githubusercontent.com/assets/19769307/21705926/12314166-d3fe-11e6-8809-bf00f4acecf9.PNG)

Is there anyone who can help me ? Thanks a lot.

**Comment by weiliu89** [comment]
Please follow README.md to create the dataset first.

**Comment by 19931991** [comment]
Ｄo you have solved the problem , and I meet it too , can you tell me the method ? my data of lmdb is :
![image](https://user-images.githubusercontent.com/28551961/34565768-13256388-f197-11e7-9924-2b33888e1732.png)..and the size of train data is 73 bytes , the size of test data is 69 bytes .

and my location of data in train.py is 
# The database file for training data. Created by data/text/create_data.sh
train_data = "/home/neu/gzl/TextBoxes-TextBoxes/data/VOC_gzl/lmdb/VOC_gzl_trainval_lmdb"
# The database file for testing data. Created by data/text_10x/create_data.sh
test_data = "/home/neu/gzl/TextBoxes-TextBoxes/data/VOC_gzl/lmdb/VOC_gzl_test_lmdb"

Wish for your reply ! Thank you very much ! @weiliu89 @wendaoyuchen

**Comment by 19931991** [comment]
Do you haved solved the problem? I meet it , and can you tell me the method ? Thank you very much ! 
My data of lmdb is : 
![image](https://user-images.githubusercontent.com/28551961/34593787-fc67b09e-f207-11e7-9727-8de39c452f90.png)

,and this is my data path setting in the train.py : 
![image](https://user-images.githubusercontent.com/28551961/34595762-55071d54-f215-11e7-8121-964fcfa0afef.png)
![image](https://user-images.githubusercontent.com/28551961/34595775-6fdefe94-f215-11e7-951e-b82518c63ad9.png)

and this is my path setting in train.prototxt :and test.prototxt :  

![Uploading image.png…]()
![Uploading image.png…]()

and my error is : 
![Uploading image.png…]()

Looking forward to your reply! Thank you very much !

**Comment by 19931991** [comment]
Do you haved solved the problem? I meet it , and can you tell me the method ? Thank you very much !
My data of lmdb is :
![image](https://user-images.githubusercontent.com/28551961/34595887-6cb9c8d8-f216-11e7-80f3-ad395bd2995f.png)

,and this is my data path setting in the train.py :
![image](https://user-images.githubusercontent.com/28551961/34595939-cf5734ee-f216-11e7-90df-373119ccbc1a.png)
![image](https://user-images.githubusercontent.com/28551961/34595946-de647712-f216-11e7-8f05-9d3641e82374.png)

and this is my path setting in train.prototxt :and test.prototxt : 
![image](https://user-images.githubusercontent.com/28551961/34595976-2c61610a-f217-11e7-8f3b-49b5941a9732.png)
![image](https://user-images.githubusercontent.com/28551961/34595997-51931a18-f217-11e7-99ba-e19a8cead41d.png)
![image](https://user-images.githubusercontent.com/28551961/34596006-5f874e78-f217-11e7-8646-a2d0358d2e08.png)

and my error is :
![image](https://user-images.githubusercontent.com/28551961/34596021-73b561d2-f217-11e7-9ffd-54bf93f29089.png)
Looking forward to your reply! Thank you very much !

**Comment by 19931991** [comment]
@weiliu89 @wendaoyuchen

**Comment by 19931991** [comment]
Do you haved solved the problem? I meet it , and can you tell me the method ? Thank you very much !
My data of lmdb is : 
![image](https://user-images.githubusercontent.com/28551961/34596646-777aa552-f21c-11e7-9e98-2135494da07a.png)

and my train data is 73 bytes, test data is 69 bytes , my data is from icdar2013.
,and this is my data path setting in the train.py :
![image](https://user-images.githubusercontent.com/28551961/34596662-99eb7076-f21c-11e7-913c-5b46c01ef361.png)
![image](https://user-images.githubusercontent.com/28551961/34596699-e0a9cd50-f21c-11e7-8e86-2c8a0897d40b.png)

and this is my path setting in train.prototxt :and test.prototxt :
![image](https://user-images.githubusercontent.com/28551961/34596713-fd1c0278-f21c-11e7-9f23-d4a9620bf2a2.png)
![image](https://user-images.githubusercontent.com/28551961/34596722-07c8caee-f21d-11e7-8bcb-551f8081bcc9.png)

![image](https://user-images.githubusercontent.com/28551961/34596733-169474d8-f21d-11e7-843f-fa11d5809cef.png)

\and my error is :
![image](https://user-images.githubusercontent.com/28551961/34596838-e0f32ff8-f21d-11e7-814f-ede2e90ae913.png)
Looking forward to your reply! Thank you very much  @wendaoyuchen   @weiliu89

**Comment by zeehasham** [comment]
I am getting the same error on VOC data. Did you find the fix? Thanks

**Comment by 19931991** [comment]
No,I I have not solved this problem，and I created the lmdb on another computer!

**Comment by eai2x** [comment]
I got this same error,can you help me?  another computer? it use this project to make?

**Comment by jaysimon** [comment]
I solved this problem, by this way:
```
ln -s $your_data$/lmdb/VOC0712_trainval_lmdb $caffe_root$/examples/VOC0712/VOC0712_trainval_lmdb
ln -s $your_data$/lmdb/VOC0712_test_lmdb $caffe_root$/examples/VOC0712/VOC0712_test_lmdb
```
It's because that, caffe cannot find lmdb data.
